### DISCUSSION TITLE
react diagram

### DISCUSSION THREAD

**Discussion by suhabe** [markdoc/markdoc#359] [Q&A] [answered]
Hello,

Where can I find documentation on how to use the "diagram", "node", "edge" tags show at the markdoc blog post to generate the flowchart visualization: https://stripe.com/blog/markdoc?

**Comment by rpaul-stripe** [comment, marked as answer]
The diagram tags demonstrated in that blog post are an internal feature that we developed at Stripe for our own documentation. Those tags are not part of Markdoc itself and we haven't published them publicly. The implementation is somewhat Stripe-specific at this time, but it's possible we may share something like it at some point in the future.

The underlying React components that display the interactive diagrams are powered by [React Flow](https://reactflow.dev/), so you could build your own tags that wrap React Flow if you wanted to.
